java-frontend/src/test/java/org/sonar/java/bytecode/se/BytecodeEGWalkerExecuteTest.java

```diff
@@ -926,4 +932,63 @@ private static ProgramState setDoubleOrLong(ProgramState programState, SymbolicV
       return programState.removeConstraintsOnDomain(sv, BytecodeEGWalker.StackValueCategoryConstraint.class);
     }
   }
+
+  @Test
+  public void test_enqueuing_only_happy_path() {
```

Comment:
this test file is already quite big and these tests have different focus and structure, I think it would be better to have them in a new file